src/Form/QuestionType/QuestionTypeDropdown.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Form\QuestionType;
+
+use Glpi\Application\View\TemplateRenderer;
+use Glpi\Form\Question;
+use Override;
+
+final class QuestionTypeDropdown extends AbstractQuestionTypeSelectable
+{
+    #[Override]
+    public function getInputType(?Question $question): string
+    {
+        return $this->isMultipleDropdown($question) ? 'checkbox' : 'radio';
+    }
+
+    #[Override]
+    public function getCategory(): QuestionTypeCategory
+    {
+        return QuestionTypeCategory::DROPDOWN;
+    }
+
+    /**
+     * Check if the question allows multiple options to be selected
+     *
+     * @param ?Question $question
+     * @return bool
+     */
+    public function isMultipleDropdown(?Question $question): bool
+    {
+        if ($question === null) {
+            return false;
+        }
+
+        return $question->getExtraDatas()['is_multiple_dropdown'] ?? false;
+    }
+
+    #[Override]
+    public function hideOptionsContainerWhenUnfocused(): bool
+    {
+        return true;
+    }
+
+    #[Override]
+    public function loadJavascriptFiles(): array
+    {
+        return array_merge(
+            parent::loadJavascriptFiles(),
+            ['js/form_question_dropdown.js']
+        );
+    }
+
+    #[Override]
+    public function getFooterScript(): string
```

Comment:
I think it is not publicly used.
```suggestion
    protected function getFooterScript(): string
```